docs/pages/demo.rst

```diff
@@ -242,7 +365,9 @@ For example, we can change the PID gains of the thermostat:
 
 ..  code-block:: sh
 
-    y r thermostat.pid.gains 2 0 0
+    y r 42 thermostat.pid.gains       # read current values
+    y r 42 thermostat.pid.gains 2 0 0 # write new values
+    y r 42 thermostat.pid.gains       # check values
```

Comment:
The last one is redundant because the new values are returned after the write call.